packages/commonjs-shim/src/module.ts

```diff
@@ -0,0 +1,16 @@
+import type { Plugin } from 'rollup';
+
+import { transformCJSToESMSyntax } from './utils';
```

Comment:
```suggestion
import { shimEsm } from './utils';
```

just a suggestion, bc I prefer succinct fn names.

Reply:
done.